### DISCUSSION TITLE
need help in creating WorkflowClient instance

### DISCUSSION THREAD

**Discussion by laxmikantb** [Netflix/conductor#2619] [Q&A]
I am trying to create bean as - in spring boot app
    @Bean
    public WorkflowClient workflowClient() {
        WorkflowClient client = new WorkflowClient();
        client.setRootURI(url);
        return client;
    }

but it fails with Caused by: java.lang.NoClassDefFoundError: javax/ws/rs/core/NoContentException
	at com.netflix.conductor.client.http.ClientBase.<init>(ClientBase.java:88) ~[conductor-client-3.3.6.jar:3.3.6]
	at com.netflix.conductor.client.http.WorkflowClient.<init>(WorkflowClient.java:92) ~[conductor-client-3.3.6.jar:3.3.6]

any help will be appreciated

**Comment by lzslim** [comment]
meet same error, any suggestion?

**Reply by AIWithHitesh** [reply]
I was able to create it using the above configuration
 @Bean
	public TaskClient taskClient(@Value("${conductor.server.url}") String conductorServerURL) {
		TaskClient taskClient = new TaskClient();
		taskClient.setRootURI(conductorServerURL);
		return taskClient;
	}

But, I think I had to add these two dependencies


                 <dependency>
			<groupId>org.apache.httpcomponents</groupId>
			<artifactId>httpclient</artifactId>
		</dependency>
		<dependency>
			<groupId>org.glassfish.jersey.core</groupId>
			<artifactId>jersey-common</artifactId>
		</dependency>


But if it still fails, try adding one more dependency.

           <dependency>
			<groupId>javax.ws.rs</groupId>
			<artifactId>javax.ws.rs-api</artifactId>
			<version>2.1.1</version>
		</dependency>

**Reply by lzslim** [reply]
thx, I would try this
